python/py-spyder-devel/Portfile

```diff
@@ -2,22 +2,21 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           python 1.0
 PortGroup           active_variants 1.1
+PortGroup           select 1.0
 
-github.setup        spyder-ide spyder 3.2.8 v
+github.setup        spyder-ide spyder e9a2483
```

Comment:
This changes the version numbering scheme so the epoch needs to be bumped.

Reply:
okay, so for the -devel port I should update that with every revision to the current date? The thing I don't understand though is that with the current version of the Portfile, ```port``` realizes that there is an update to py-spyder-devel so it doesn't seem to matter... Can I not just delete the whole thing - what am I missing here? Related to this, in the py-spyder Portfile since the ```updated to a version is numerically less than the previous version``` does not apply there, can I remove the ```epoch``` line?
Thanks!

Reply:
The git tag should never be used for the version. Manually set the version to the devel version and date. For example, 3.2.8-20180522. The gimp3-devel port uses that version numbering structure. 

No need for an epoch line in that case too.